ethereum/core/src/test-support/java/org/hyperledger/besu/ethereum/core/TransactionTestFixture.java

```diff
@@ -30,7 +30,7 @@
 
   private long nonce = 0;
 
-  private Wei gasPrice = Wei.of(5);
+  private Wei gasPrice = Wei.of(5000);
```

Comment:
I am surprised this didn't cause more test fallout ☘️

Reply:
Looks like the places that care about it set it explicitly!